sdk/examples/xo_python/sawtooth_xo/xo_cli.py

```diff
@@ -170,10 +176,16 @@ def do_init(args, config):
     if args.url is not None:
         url = args.url
 
+    user = config.get('DEFAULT', 'user')
+    if args.user is not None:
+        user = args.user
+
     config.set('DEFAULT', 'username', username)
     print("set username: {}".format(username))
     config.set('DEFAULT', 'url', url)
     print("set url: {}".format(url))
+    config.set('DEFAULT', 'user', user)
```

Comment:
We should move the --user argument to all sub-commands instead of during init to avoid persisting a plaintext password.